test/formatters/sarif-test.js

```diff
@@ -0,0 +1,301 @@
+            {
+              physicalLocation: {
+                artifactLocation: {
+                  uri: expect.stringMatching('file://.*/app/templates/application.hbs'),
+                  index: 0,
+                },
+                region: {
+                  startLine: 1,
+                  startColumn: 4,
+                  snippet: {
+                    text: 'Here too!!',
+                  },
+                },
+              },
+            },
+          ],
+          ruleId: 'no-bare-strings',
+          ruleIndex: 0,
+        },
+        {
+          level: 'error',
+          message: {
+            text: 'Non-translated string used',
+          },
+          locations: [
+            {
+              physicalLocation: {
+                artifactLocation: {
+                  uri: expect.stringMatching('file://.*/app/templates/application.hbs'),
+                  index: 0,
+                },
+                region: {
+                  startLine: 1,
+                  startColumn: 24,
+                  snippet: {
+                    text: 'Bare strings are bad...',
+                  },
+                },
+              },
+            },
+          ],
+          ruleId: 'no-bare-strings',
+          ruleIndex: 0,
+        },
+        {
+          level: 'warning',
+          message: {
+            text: 'HTML comment detected',
+          },
+          locations: [
+            {
+              physicalLocation: {
+                artifactLocation: {
+                  uri: expect.stringMatching('file://.*/app/templates/application.hbs'),
+                  index: 0,
+                },
+                region: {
+                  startLine: 1,
+                  startColumn: 53,
+                  snippet: {
+                    text: '<!-- bad html comment! -->',
+                  },
+                },
+              },
+            },
+          ],
+          ruleId: 'no-html-comments',
+          ruleIndex: 1,
+        },
+      ],
+    },
+  ],
+};
+
+const ROOT = process.cwd();
+const DEFAULT_OPTIONS = {
+  console: undefined,
+  includeTodo: false,
+  isInteractive: true,
+  outputFile: undefined,
+  quiet: false,
+  updateTodo: false,
+  verbose: false,
+  workingDir: undefined,
+};
+
+describe('', () => {
+  let project;
+
+  beforeEach(function () {
+    project = Project.defaultSetup();
+    project.chdir();
+  });
+
+  afterEach(async function () {
+    process.chdir(ROOT);
+    await project.dispose();
+  });
+
+  it('should output sarif log to default path (in project working directory)', function () {
+    let sarifWindowsPattern = /Report\swritten\sto\s(.*\\ember-template-lint-report-\d{4}(?:-\d{2}){3}(?:_\d{2}){2}\.sarif)/;
+    let sarifUnixPattern = /Report\swrit{2}en\sto\s(.*\/ember-template-lint-report-\d{4}(?:-\d{2}){3}(?:_\d{2}){2}\.sarif)/;
+    let pattern = IS_WINDOWS ? sarifWindowsPattern : sarifUnixPattern;
+    let formatter = new Sarif(
+      Object.assign(DEFAULT_OPTIONS, {
```

Comment:
Can you throw another `{}` in here? As it stands, you currently mutate `DEFAULT_OPTIONS` but don't "clean it up" in teardown.

```suggestion
      Object.assign({}, DEFAULT_OPTIONS, {
```

something like that (don't actually accept, it won't pass prettier)

Reply:
Yep.

Reply:
Done.